src/App.tsx

```diff
@@ -67,29 +67,27 @@ dayjs.updateLocale('en', {
 const App = () => {
   const [theme, setTheme] = useState<DefaultTheme>(themes.light)
 
-  useEffect(() => {
-    return store.subscribe(() => {
-      setTheme(themes[store.getState().settings.theme])
-    })
-  }, [])
+  useEffect(
+    () =>
+      store.subscribe(() => {
+        setTheme(themes[store.getState().settings.theme])
+      }),
+    []
+  )
 
   return (
-    <RootSiblingParent>
-      <SafeAreaProvider>
-        <Provider store={store}>
-          <Main>
-            <ThemeProvider theme={theme}>
-              <RootStackNavigation />
-              <StatusBar style={theme.name === 'light' ? 'dark' : 'light'} />
-            </ThemeProvider>
-          </Main>
-        </Provider>
-      </SafeAreaProvider>
-    </RootSiblingParent>
+    <Provider store={store}>
+      <Main>
+        <ThemeProvider theme={theme}>
+          <RootStackNavigation />
+          <StatusBar style={theme.name === 'light' ? 'dark' : 'light'} />
+        </ThemeProvider>
+      </Main>
+    </Provider>
   )
 }
 
-const Main = ({ children }: { children: ReactNode }) => {
+const Main: FC = ({ children }) => {
```

Comment:
Nit: I seem to remember that we decided not to rely on `FC` anymore, like [many others](https://backstage.io/docs/architecture-decisions/adrs-adr006/#:~:text=Facebook%20has%20removed%20React.,combination%20with%20a%20few%20downsides.).

Reply:
For clarification, this comes from here: https://github.com/alephium/mobile-wallet/blob/swap/src/types/components.d.ts#L19

Last thing I remember was that we use `FC` when we need the `children` prop to be there, and not when we don't. But that was for the desktop wallet, I don't remember if we had discussed this on the mw side.

I have not encountered any disadvantages using `FC`. If anything, it makes our code-bases more consistent. We don't need to switch the mental model in our brain that says "when I want to use `children` I just need to use `FC`.

I look forward to your thoughts!

Reply:
https://github.com/facebook/create-react-app/pull/8177

Reply:
I personally like that we type the children prop manually. This makes it very explicit that we're actually expecting children. It also allows us to be more strict when we need to, ie. accept only children of a certain type, accept only 1 child, not 2 etc.

And it's funny, because for me using FC makes our component declaration less consistent. Sometimes we use it, sometimes we don't. I like when we always have the full list of typed props defined just above the component itself. 😊

Reply:
If this is fine by you, then yeah! 🚀